RecoLocalCalo/HcalRecProducers/src/ZdcHitReconstructor.cc

```diff
@@ -36,6 +36,7 @@ ZdcHitReconstructor::ZdcHitReconstructor(edm::ParameterSet const& conf):
 { 
   tok_input_hcal = consumes<ZDCDigiCollection>(conf.getParameter<edm::InputTag>("digiLabelhcal"));
   tok_input_castor = consumes<ZDCDigiCollection>(conf.getParameter<edm::InputTag>("digiLabelcastor"));
+  tok_input_qie10 = consumes<QIE10DigiCollection>(conf.getParameter<edm::InputTag>("digiLabelQIE10ZDC"));
```

Comment:
I do not see a point to put this hook in place now without actual code consuming the digis.